scripts/clean-rookd.sh

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 sudo rm -fr /var/lib/rook
+sudo mkdir /var/lib/rook;sudo chown -R $USER:$USER /var/lib/rook
```

Comment:
why create the folder after deleting it? is this so it's simpler to start the cluster on the next run?